server/websocket_fallback.py

```
@@ -156,81 +174,140 @@ def _handle_client(self, client, addr, handler):
-                                client.sendall(self._encode_frame(msg))
-
-                            res = handler(message, send_func)
-                            if inspect.isawaitable(res):
-                                asyncio.run(res)
-
-                    # Read more data if buffer doesn't have a full frame
-                    data = client.recv(2048)
-                    if not data:
-                        break
-                    buffer += data
+                        except Exception:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)